posthog/models/action/action.py

```diff
@@ -72,24 +72,8 @@ def get_analytics_metadata(self):
 
     @property
     def steps(self) -> list[ActionStepJSON]:
-        if self.steps_json is None:
```

Comment:
It looks like at this point there are no more actions with `steps_json` being null in either US or EU. Can we make the field `steps_json` to enforce this for safety? This should be almost instant to execute

Reply:
Make sense. I'll do it as a one off follow up if thats alright